sources/packages/backend/apps/api/src/constants/error-code.constants.ts

```diff
@@ -121,3 +121,9 @@ export const DUPLICATE_INSTITUTION_LOCATION_CODE =
  * Duplicate SABC code for the institution.
  */
 export const DUPLICATE_SABC_CODE = "DUPLICATE_SABC_CODE";
+
+export const FIRST_COE_NOT_COMPLETE_MESSAGE =
```

Comment:
so are we planning to keep these error messages here?
```
export const FIRST_COE_NOT_COMPLETE_MESSAGE =

export const INVALID_TUITION_REMITTANCE_AMOUNT_MESSAGE =
```

Reply:
Forgot to remove those constant. Will remove.